.dockerignore

```diff
@@ -4,6 +4,9 @@
 config
 docs
 
+# asdf
```

Comment:
I feel kinda weird editing a root file, I could drop this if you'd rather

Reply:
Please make a separate PR for this if it's needed.

Reply:
Removed from this pr